src/sas/qtgui/MainWindow/GuiManager.py

```diff
@@ -194,7 +194,7 @@ def loadAllPerspectives(self):
                 loaded_perspective = perspective(parent=self)
                 loaded_dict[name] = loaded_perspective
             except Exception as e:
-                logger.log(f"Unable to load {name} perspective.\n{e}")
+                logger.warning(f"Unable to load {name} perspective.\n{e}")
```

Comment:
I would claim this warrants `error` or even `critical` level. However, seeing that the default log level for sasview is `warning`, this is OK, since it will be shown.